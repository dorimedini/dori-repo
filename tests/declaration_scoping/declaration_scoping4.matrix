{
	matrix x(3,2);
	while (x[1,1]==0) 
	{
		print x;
		x[1,1] = 2;
		int x=3;
	}
	print x;
	int y=8;
	{
		matrix x(3,2) = [2,3;0,0;y,y];
		print x;
	}
	{
		print x;
		int x=8+y;
		print x;
	}
	x[1,1]=9;
	print x;
}