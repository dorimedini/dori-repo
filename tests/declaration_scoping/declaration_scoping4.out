[0,0;0,0;0,0]
[0,0;0,2;0,0]
[2,3;0,0;8,8]
[0,0;0,2;0,0]
16
[0,0;0,9;0,0]
